---
title: MSSQLSERVER_26013 | Документация Майкрософт
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: supportability
ms.topic: language-reference
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
author: MashaMSFT
ms.author: mathoma
ms.openlocfilehash: 19e9bd114135d2df7694dd7d7fba60287aa5d519
ms.sourcegitcommit: da88320c474c1c9124574f90d549c50ee3387b4c
ms.translationtype: HT
ms.contentlocale: ru-RU
ms.lasthandoff: 07/01/2020
ms.locfileid: "85723782"
---
# <a name="mssqlserver_26013"></a>MSSQLSERVER_26013
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
  
## <a name="details"></a>Сведения  
  
| attribute | Значение |  
| :-------- | :---- |  
|Название продукта|SQL Server|  
|Идентификатор события|26013|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|SNI_SSL_READY|  
|Текст сообщения|Сертификат [Cert Hash(sha1) "%hs"] был успешно загружен для шифрования.|  
  
## <a name="explanation"></a>Объяснение  
Это информационное сообщение. Указанный сертификат был успешно загружен в [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] и может использоваться для шифрования.  
  
## <a name="user-action"></a>Действие пользователя  
Никаких действий не требуется.  
  
